diaries.html

```diff
@@ -462,9 +468,11 @@ <h4>Fun Blast</h4>
                     </div>
                 </div>
 
-                <div class="freshbits-diaries-row">
+                <div class="freshbits-diaries-row" id="learning-hour-3">

```

Comment:
learning-hour -> learning-hours

No need to add month and year in learning hours